runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/MonitoringUtil.java

```diff
@@ -180,14 +180,15 @@ public int compare(JobMessage o1, JobMessage o2) {
     return allMessages;
   }
 
-  public static String getJobMonitoringPageURL(String projectName, String jobId) {
+  public static String getJobMonitoringPageURL(String projectName, String regionId, String jobId) {
```

Comment:
I believe this is a backward incompatible change. Someone from the Java team should verify that this interface is not meant to be used by the end users.